virttest/test_setup.py

```diff
@@ -577,16 +577,58 @@ def get_multi_supported_hugepage_size(self):
             raise ValueError("Root hugepage control sysfs directory %s did not"
                              " exist" % self.pool_path)
 
-    def get_node_num_huge_pages(self, node, pagesize):
+    def get_kernel_hugepages(self, pagesize):
```

Comment:
please avoid redundant code by abstracting methods `get_node_num_huge_pages()` and `set_node_num_huge_pages()` with your requirement of different sysfs path.

Reply:
@balamuruhans you mean merge the get_node_num_huge_pages and set_node_num_huge_pages? Due to the code have been used before, it maybe affect some existed cases in tp-libvirt and tp-qemu

Reply:
no we should always maintain backward compatibility, I meant to reuse `kernel_interface.py` APIs
```python
def func(self, node_path, value=None):
    hp = kernel_interface.SysFS(node_path)
    if value:
        hp.sys_fs_value = value
        return hp.sys_fs_value == str(value)
     return hp.sys_fs_value
def existing_func(self): # to maintain compatibility
     node_path = "/sys/devices/system/node//hugepages/hugepages-%skB/nr_hugepages"
     return func(node_path)
def new_func(self):
     mem_path = "/sys/kernel/mm/hugepages/hugepages-${pagesize}kB/nr_hugepages"
     return func(mem_path)
```

Reply:
@balamuruhans Updated, please help review again